GOVERNANCE.md

```diff
@@ -146,8 +146,31 @@ Contributions can be:
 
 ### Nominating a new Collaborator
 
-To nominate a new Collaborator, open an issue in the [nodejs/node][] repository.
-Provide a summary of the nominee's contributions. For example:
+To nominate a new Collaborator:
+
+1. **Optional but strongly recommended**: open a
+   [discussion in the nodejs/collaborators][] repository. Provide a summary of
+   the nominee's contributions (see below for an example).
```

Comment:
The collaborators team contains 86 people but only 49 are watching the collaborators repo, which is 57%, so a lot of them would not receive a notification in the feedback stage. So I would recommend pinging the team in the private discussion.
```suggestion
   [discussion in the nodejs/collaborators][] repository. Provide a summary of
   the nominee's contributions (see below for an example). Mention
   @nodejs/collaborators in the discussion to notify other collaborators about
   the nomination.
```

Reply:
I'm strongly -1 on that, if folks chose to not receive notifications, we should not force it – it will only encourage them to also unsubscribe from notifications to team mention

Reply:
Perhaps we can add an optional-but-recommended subscription to the collaborators repo somewhere? For example, in https://github.com/nodejs/node/blob/36e89dd13cdf7d5a81e0d4b53c0070dd1790705e/onboarding.md?plain=1#L51

Reply:
Mentioning it in the collaborator onboarding guide might be a nice solution: https://github.com/nodejs/node/pull/57527